src/services/navigationBar.ts

```diff
@@ -269,21 +288,73 @@ namespace ts.NavigationBar {
                 addLeafNode(node);
                 break;
 
+            case SyntaxKind.CallExpression:
             case SyntaxKind.BinaryExpression: {
                 const special = getAssignmentDeclarationKind(node as BinaryExpression);
                 switch (special) {
                     case AssignmentDeclarationKind.ExportsProperty:
                     case AssignmentDeclarationKind.ModuleExports:
-                    case AssignmentDeclarationKind.PrototypeProperty:
-                    case AssignmentDeclarationKind.Prototype:
                         addNodeWithRecursiveChild(node, (node as BinaryExpression).right);
                         return;
+                    case AssignmentDeclarationKind.PrototypeProperty:
+                    case AssignmentDeclarationKind.Prototype: {
+                        const binaryExpression = (node as BinaryExpression);
+                        const assignmentTarget = binaryExpression.left as PropertyAccessExpression;
+                        if (isPropertyAccessExpression(assignmentTarget.expression)) {

```

Comment:
Is anything being done for `AssignmentDeclarationKind.Prototype` now?

Reply:
@jessetrinity It should go back to the default, although there is a slight change in behavior. Previously there would be an `unknown` node inserted as the parent of anything defined in the `prototype` and that node is missing now. I will look into getting the old behavior back.

Reply:
As long as we still get the `unknown` node it seems fine to me to leave prototype assignments as a TODO item.

Reply:
I played around with it a bit more. Breaking out the `AssignmentDeclarationKind.Prototype` case and adding the `ObjectLiteralExpression` properties to that node seems to do the trick. Don't need the `prototypeAccess` either since we should be in one...

```ts
case AssignmentDeclarationKind.Prototype: {
    const binaryExpression = (node as BinaryExpression);
    const assignmentTarget = binaryExpression.left as PropertyAccessExpression;

    // If we see a prototype assignment, start tracking the target as a class
    if (isIdentifier(assignmentTarget.expression)) {
        addTrackedEs5Class(assignmentTarget.expression.text);
    }
    if (isFunctionExpression(binaryExpression.right) || isArrowFunction(binaryExpression.right)) {
        addNodeWithRecursiveChild(node,
            binaryExpression.right,
            assignmentTarget.expression as Identifier);
    }
    else {
        startNode(binaryExpression, assignmentTarget.expression as Identifier);
            forEachChild(binaryExpression.right, addChildrenRecursively);
        endNode();
    }
    return;
}
```

I had to set `[AssignmentDeclarationKind.Prototype]: true` in `isEs5ClassMember`. I haven't tested it thoroughly enough to see if that breaks anything else. It does look like children of the constructor are showing up in the wrong place though (directly under the class). It's otherwise looking really good, super stoked for your changes!

![lookingGood](https://user-images.githubusercontent.com/42591254/63644050-becbd680-c694-11e9-8bee-54e1dce8d4ca.png)

Reply:
@jessetrinity Yeah, i would expect it works ok. I regretted reverting back to `unknown` almost as soon as pushed 😅. I will integrate the `prototype` assignment. I will also fix nested assignments which are the cause of the new lines while editing.

Reply:
@jessetrinity I changed the code to support object literal prototype assignment.